tests/test_client.py

```diff
@@ -12,11 +12,26 @@ def setup_method(self):
         self.api_key = "super_secret_api_key"
         self.user_agent = "fintoc-python/test"
         self.params = {"first_param": "first_value", "second_param": "second_value"}
+        self.version = "2023-01-01"
 
-    def create_client(self, params=False):
-        if not params:
+    def create_client(self, params=False, version=False):
```

Comment:
Acá version debería ser `None`, y de ahí revisar `if version is None`

Reply:
jaja no había cachado que habías comentado e hice rebase :shame:. Lo cambié a `None` y al final no fue necesario el `if`, se lo paso siempre no más y quedó el `if` del params solamente